docs/api/user.md

```diff
@@ -9,9 +9,7 @@ Base endpoint:
 {server_url}/users
 ```
 
-Contains information about the users of the service.
-
-To have a task in the service, the user must be added to
+Contains information about the users of the service. To have a task in the service, the user must be added to
```

Comment:
I try to start new sentences on a new line to make future revisions easier to spot. Changes to the 2nd, 3rd, 4th... sentence are easier to spot and compare when the lines don't wrap in the display.